packages/react-components/react-card/src/components/Card/useCardStyles.styles.ts

```diff
@@ -25,9 +26,10 @@ export const cardCSSVars = {
   cardBorderRadiusVar: '--fui-Card--border-radius',
 };
 
-const focusOutlineStyle = {
+const focusOutlineStyle: Partial<FocusOutlineStyleOptions> = {
   outlineRadius: `var(${cardCSSVars.cardBorderRadiusVar})`,
   outlineWidth: tokens.strokeWidthThick,
+  outlineOffset: '-2px', // FIXME: tokens.strokeWidthThick causes some weird bugs
```

Comment:
This uses the same hardcoded value as the [createFocusOutlineStyle.ts](https://github.com/microsoft/fluentui/pull/27966/files#diff-5725b88e9f70bf2716bbb7d17237b032b555e693e32e5221b86b9e8c5026e9b2)

Reply:
Is there a bug reported for this weird behavior?

Reply:
Not that I'm aware, but I only copied the same value used on the createFocusOutlineStyle.ts file.